README.md

```diff
@@ -170,6 +173,7 @@ const optionalConfigObject = {
   sensorDescription: "Touch sensor", // Android
   sensorErrorDescription: "Failed", // Android
   cancelText: "Cancel", // Android
+  backgroundMode: true, //Android
```

Comment:
I would rather keep the `backgroundMode` as `false` in the example. That's the first code which developer simply copy&paste to his codebase to see how the plugin works and it would be better if he sees anything (popup) comparing to the background mode.

Also after comment there should be a space `//Android` into `// Android`